iib/web/migrations/versions/49d13af4b328_add_add_deprecations_api_endpoint.py

```diff
@@ -0,0 +1,162 @@
+"""Add add-deprecations API endpoint.
+
+Revision ID: 49d13af4b328
+Revises: 1920ad83d0ab
+Create Date: 2024-07-26 00:17:44.283197
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '49d13af4b328'
+down_revision = '1920ad83d0ab'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'deprecation_schema',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('schema', sa.Text(), nullable=False),
+        sa.PrimaryKeyConstraint('id'),
+    )
+    with op.batch_alter_table('deprecation_schema', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_deprecation_schema_schema'), ['schema'], unique=True)
```

Comment:
Is there a reason to name the index `ix_deprecation_schema_schema` instead of `ix_deprecation_schema`?